tests/backend/core/Core/Unit/Logic/PermissionsTest.php

```diff
@@ -3102,7 +3102,6 @@ public function permissionsTests(): array
                     'feature_procedure_get_base_data',
                     'feature_procedure_single_document_upload_zip',
 //                  'feature_statement_data_input_orga', //fixme: permission is not set and it is unclear why this line is here - area_statement_data_input_orga instead is set - might be a mismatch
-                    'field_statement_public_allowed',
```

Comment:
you may add this permission to the `featuresDenied` area so that it is tested to be denied

Reply:
[7ff2c1b](https://github.com/demos-europe/demosplan-core/pull/1738/commits/7ff2c1bdfd38a6595b1fb39ea852d8bc67eb2d9e)